### PR TITLE
chore: sync en

### PR DESCRIPTION
**Pull request by mroz22**

### CODE REVIEW

File: packages/suite-data/files/translations/en.json [hunk 1]
```diff
@@ -837,7 +837,6 @@
     "TR_LEARN_ADVANCED_RECOVERY": "Learn how to enter letters",
     "TR_LEARN_AND_DISCOVER": "Suite Guide",
     "TR_LEARN_MORE": "Learn more",
-    "TR_LEARN": "Learn",
```
Comment (dimasivan) [inline packages/suite-data/files/translations/en.json:840]: this translation is in use https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/views/guide/OpenGuideFromTooltip.tsx#L42

Comment (mroz22) [reply packages/suite-data/files/translations/en.json:840]: Ah, I see. Check the last commit. Seems some translations were not synced with crowdin yet?

Comment (dimasivan) [reply packages/suite-data/files/translations/en.json:840]: tru, fixed in b2e5bf8

**Review by matejkriz — APPROVED** [review]
LGTM

**Pull request merged**